heareval/tasks/sampler.py

```diff
@@ -45,7 +45,7 @@
     },
     "dcase2016_task2": {
         "task_config": dcase2016_task2_config,
-        "audio_sample_size": 10,
+        "audio_sample_size": 4,
```

Comment:
Decrease the number of files in the audio sample size for dcase dataset so that the file is less than 25 mb so that it can be in the github repo.